server/e2e_test_command.go

```diff
@@ -24,8 +24,19 @@ const (
 	e2eTestMsgOpts    = "Triggering E2E testing with options:"
 	e2eTestFmtOpts    = "%v\n```%v```"
 	e2eTestFmtSuccess = "Successfully triggered E2E testing!\n[GitLab pipeline](%v) | [Test dashboard](%v/cycle/%v)"
+	serverTypeFlag    = "--server-type"
+	cloudServerType   = "cloud"
 )
 
+var cloudEnvOpts = map[string]string{
+	"NOTIFY_ADMIN_COOL_OFF_DAYS":         "0.00000001",
```

Comment:
@yasserfaraazkhan 👀

Reply:
I need to add `SERVER_TYPE:cloud` here right?

Reply:
Yes, here we need a var `SERVER_TYPE: "cloud"`

I'll confirm if we need the other variables.

@saturninoabril @metanerd 
https://git.internal.mattermost.com/qa/cypress-ui-automation/-/merge_requests/53/diffs

in above PR, I am  basically importing the Scripts present in the other working cloud branch.
making settings according to flag was getting extremely complicated.

now we do not need to handle Variable/Settings overriding from mattermod....

we just need to send SERVER_TYPE=(cloud || ha || any-other) and this will trigger the corresponding .gitlab-ci-SERVER.yml file











I'm testing it still ...Not sure if this will break something

Reply:
Like what Yasser said, these variables should be handled in the GitLab runner and not here in mattermod. If server type (`--server-type`) is present and with valid values (any of `single`, `cloud`, `ha` or `subpath`) then `SERVER_TYPE` should be included as variables to trigger GitLab pipeline.

Reply:
@AGMETEOR you are correct, we can do it with `e2e-test SERVER_TYPE=ha`
and based on value, I can see other things to handle on gitlab.yml.

@saturninoabril , let us know if we are thinking of same approach